### PR TITLE
Updated p4rt-ctl.rst doc for PacketModMeter CLI

### PR DESCRIPTION
**Pull request by bharticemk**
Updated p4rt-ctl.rst doc for PacketModMeter CLI
 - Add configuration to meter table
 - Add table entry (rule) for Direct meter
 - Get direct meter value
 - Get indirect meter value

### CODE REVIEW

**Review by satish153 — APPROVED** [review]

File: docs/clients/p4rt-ctl.rst [hunk 1]
```diff
@@ -469,6 +469,86 @@ Example:
...
+
+.. code-block:: bash
+
+   add-meter-config SWITCH METER_TBL METER_CONFIGURATION
+
+Arguments:
+
+* ``SWITCH``: Bridge name. Maps internally to device name.
+* ``METER_TBL``: Meter table.
+* ``METER_CONFIGURATION``: COnfiguration for meter table.
```
Comment (ffoulkes) [inline docs/clients/p4rt-ctl.rst:483]:
```suggestion
* ``METER_CONFIGURATION``: Configuration for meter table.
```

File: docs/clients/p4rt-ctl.rst [hunk 2]
```diff
@@ -469,6 +469,86 @@ Example:
...
+
+.. code-block:: bash
+
+   p4rt-ctl get-packet-mod-meter SWITCH METER_TABLE COUNTER_FLOW
+
+Arguments:
+
+* ``SWITCH``: Bridge name. Maps internally to device name.
+* ``METER_TABLE``: Specifies meter table entry from the p4 file.
+* ``METER_FLOW``: Meter ID (generated ID by p4c; see bfrt.json file)
```
Comment (ffoulkes) [inline docs/clients/p4rt-ctl.rst:542]: The argument in the model command is `COUNTER_FLOW`. Which one is correct?

**Review by ffoulkes — APPROVED** [review]
One typo, one mismatch in the name of an argument.

Approved. Please correct before merging.

File: docs/clients/p4rt-ctl.rst [hunk 3]
```diff
@@ -469,6 +469,86 @@ Example:
...
+* ``METER_TBL``: Meter table.
+* ``METER_CONFIGURATION``: COnfiguration for meter table.
+
+Example:
+
+.. code-block:: bash
+
+   p4rt-ctl add-meter-config br0 my_control.meter1 "meter_id=2244878476,meter_index=10,meter_config=policer_meter_prof_id=0,policer_spec_cir_unit=1,policer_spec_cbs_unit=1,policer_spec_eir_unit=1,policer_spec_ebs_unit=1,policer_spec_cir=1000,policer_spec_cbs=1500,policer_spec_eir=1000,policer_spec_ebs=1500"
+
+Add table entry (rule) for Direct meter
```
Comment (5abeel) [inline docs/clients/p4rt-ctl.rst:491]:
```suggestion
Add table entry (rule) for direct meter
```
The uppercase `D` here stands out in an odd way. Other things like indirect meter in this document are not capitalized, so should switch to lowercase d.

File: docs/clients/p4rt-ctl.rst [hunk 4]
```diff
@@ -469,6 +469,86 @@ Example:
...
+.. code-block:: bash
+
+   p4rt-ctl get-direct-meter br0 my_control.i_fwd "hdrs.mac[vmeta.common.depth].da="0x000000000193",hdrs.mac[vmeta.common.depth].sa="0x9ebace99d1d2" 
+
+Get indirect meter value
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+   p4rt-ctl get-packet-mod-meter SWITCH METER_TABLE COUNTER_FLOW
```
Comment (5abeel) [inline docs/clients/p4rt-ctl.rst:536]:
```suggestion
   p4rt-ctl get-packet-mod-meter SWITCH METER_TABLE METER_FLOW
```

**Pull request merged**